azpubsub/contextvalidator/src/main/java/kafka/security/auth/ConfigUtils.java

```diff
@@ -0,0 +1,50 @@
+package azpubsub.contextvalidator.kafka.security.auth;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.HashMap;
+import java.util.Properties;
+import java.util.Map;
+
+public class ConfigUtils {
```

Comment:
we should have a AzPubSubConfig similiar to KafkaConfig that holds the configuration properties... this will load config value from properties file, override defaults, do some validation and fail if not met.

with that, use the property (i.e. class member) of AzPubSubConfig in all places.

Reply:
AzPubSubConfig should extend AbstractConfig from Kafka lib and follow the same pattern...

Reply:
@senthilm-ms I tried to rename the project like you mentioned, but the structure of the folder became very strange - all the files will be moved out of the src/main folder to another kafka.security.auth folder ..  Should I just get rid of the src/main?

Reply:
@senthilm-ms , after spending so much time trying different things, I decided not change the name of the project and the name of the package.  Otherwise I need to get rid of  src/main/java and src/main/scala folders, but we need them because we have both java/scala code.

Reply:
But I am getting rid of the default packages/empty packages.

Reply:
The idea of creating AzPubSubConfig which extends AbstractConfig is just good, but: 

- The AzPubSubConfig is unlike KakfaConfig. It will be instantiated in different place, - it is not a singleton used anywhere; 

- This is not top priority of this feature at the moment. I will first finish up tasks of high priority and come back to revisit.